plugins/woocommerce/includes/data-stores/class-wc-customer-data-store.php

```diff
@@ -328,35 +340,50 @@ private function update_user_meta( $customer ) {
 	 * @return WC_Order|false
 	 */
 	public function get_last_order( &$customer ) {
-		$last_order = apply_filters(
+		$last_order_id = apply_filters(
```

Comment:
let's add a phpcs:ignore or a docblock otherwise ci won't be happy

Reply:
👍🏻 Did both: add a docblock, but with an ignore comment for the `@since` tag.